### PR TITLE
chore: create include.gypi for Windows build again

### PR DESCRIPTION
**Pull request by rzhao271**
I'd like to add this PR to temporarily resolve the kerberos BinSkim issue on our end.
Once kerberos releases a new package, we will consume it and remove this patch to detect future upstream BinSkim issues.

### CODE REVIEW

**Review by deepak1556 — COMMENTED** [review]
LGTM, pending verification build

 https://monacotools.visualstudio.com/Monaco/_build/results?buildId=286148&view=results

File: build/azure-pipelines/win32/product-build-win32.yml [hunk 1]
```diff
@@ -89,6 +89,29 @@ steps:
...
+  - pwsh: |
+      $includes = @'
+        {
+          'target_defaults': {
+            'conditions': [
+              ['OS=="win"', {
+                'msvs_configuration_attributes': {
+                  'SpectreMitigation': 'Spectre'
+                }
+              }]
```
Comment (rzhao271) [inline build/azure-pipelines/win32/product-build-win32.yml:103]: Testing Pull Request extension

Leo vestibulum sapien donec justo nostra inceptos aenean taciti. Magna maecenas dui varius lacus consequat, blandit finibus ante. Nullam facilisi himenaeos praesent phasellus rhoncus neque leo. Litora cras augue magnis fringilla turpis nascetur aptent. Quis suspendisse orci convallis, justo curae turpis vivamus hendrerit sagittis. At arcu nulla laoreet fusce, nulla netus sit! Commodo nulla sem parturient euismod fermentum at tempus. Euismod venenatis dolor porta gravida ac senectus. Iaculis ultricies eget volutpat iaculis id varius magna aptent.

**Comment by rzhao271** [comment]
I started a newer verification build at https://monacotools.visualstudio.com/Monaco/_build/results?buildId=286190&view=results but it still has the terminal smoke test failure. Based on the pipeline runs, I believe the failure is unrelated to this change.

File: build/azure-pipelines/win32/product-build-win32.yml [hunk 2]
```diff
@@ -89,6 +89,29 @@ steps:
...
+            ]
+          }
+        }
+      '@
+
+      if (!(Test-Path "~/.gyp")) {
+        mkdir "~/.gyp"
+      }
+      echo $includes > "~/.gyp/include.gypi"
+    displayName: Create include.gypi
```
Comment (deepak1556) [inline build/azure-pipelines/win32/product-build-win32.yml:113]:
```suggestion
    displayName: Create include.gypi
    condition: and(succeeded(), ne(variables.NODE_MODULES_RESTORED, 'true'))
```

**Review by deepak1556 — COMMENTED** [review]
LGTM, with minor nit

**Review by amunger — APPROVED** [review]

**Pull request merged**